seatunnel-connectors-v2/connector-file/connector-file-base/pom.xml

```diff
@@ -93,5 +93,18 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi</artifactId>
+            <version>4.1.2</version>
+            <scope>compile</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi-ooxml</artifactId>
+            <version>4.1.2</version>
+        </dependency>
+
```

Comment:
Versions should be managed centrally in root-pom.

Reply:
Excuse me, do you know what causes this?

![image](https://user-images.githubusercontent.com/32196893/186586956-aaab7c1a-e542-4403-a2b2-77eb764e7d01.png)

Reply:
😄 Don't worry.The CodeQL is unstable